src/meta/user/userRoles.ts

```diff
@@ -42,9 +42,21 @@ const getLastRole = (user: User) => {
   return _roles[0]
 }
 
+const roleNamesOrder = [
```

Comment:
This , I would leave alphabetically sorted as well

Reply:
Or (in importance order )

Reviewer 
National corespondent 
ANC
Collaborator 
Viewer 

What do you think ?